.github/auto-merge.yml

```diff
@@ -0,0 +1,62 @@
+# Configuration for probot-auto-merge - https://github.com/bobvanderlinden/probot-auto-merge
+
+# Whether an out-of-date pull request is automatically updated.
+# It does so by merging its base on top of the head of the pull request.
+# This is the equivalent of clicking the 'Update branch' button.
+# This is useful for repositories where protected branches are used and the option
+# 'Require branches to be up to date before merging' is enabled.
+# Note: this only works when the branch of the pull request resides in the same repository as
+#       the pull request itself.
+updateBranch: true
+# Whether the pull request branch is automatically deleted.
+# This is the equivalent of clicking the 'Delete branch' button shown on merged pull requests.
+# Note: this only works when the branch of the pull request resides in the same repository as
+#       the pull request itself.
+deleteBranchAfterMerge: true
+reportStatus: true
+
+# The minimum number of reviews from each association that approve the pull request before
+# doing an automatic merge. For more information about associations see:
+# https://developer.github.com/v4/enum/commentauthorassociation/
+minApprovals:
+  COLLABORATOR: 3
```

Comment:
shouldn't we also add owner approval here?

Reply:
it shall in default, but true that it is no very clear hos the rules in root and here works together...